docs/dev/libvirt-howto.md

```diff
@@ -196,7 +196,7 @@ This step allows installer and users to resolve cluster-internal hostnames from
     ```sh
     echo server=/tt.testing/192.168.126.1 | sudo tee /etc/NetworkManager/dnsmasq.d/openshift.conf
     ```
-3. `systemctl restart NetworkManager`
+3. Tell NetworkManager to re-read the `dns` configuration change: `sudo systemctl reload NetworkManager`
```

Comment:
nit: Instead of "re-read" vs. `reload`, I'd rather just repeat "reload".  Maybe something like:

> 3. Reload NetworkManager to pick up the change: `sudo systemctl reload NetworkManager`